src/MasterSite.php

```diff
@@ -686,7 +688,16 @@ public function enqueue_public_assets() {
 			'show_scroll_times' => Search::SHOW_SCROLL_TIMES,
 		];
 
-		wp_register_script( 'main', $this->theme_dir . '/assets/build/index.js', [ 'jquery', 'cssvarsponyfill' ], $js_creation, true );
+		wp_register_script(
+			'main', $this->theme_dir . '/assets/build/index.js',
+			[
+				'jquery',
+				'cssvarsponyfill',
+				'photoswipe',
+				'photoswipeui'
+			],
+			$js_creation, true
```

Comment:
phpcs is complaining [here](https://app.circleci.com/pipelines/github/greenpeace/planet4-master-theme/1722/workflows/4e477b3c-bc89-46ce-b439-3cf09ea96a7a/jobs/9660)... 👨‍💻